server/index.js

```diff
@@ -9,13 +9,21 @@ const PORT = 3002;
 // TODO: update once static client files are available
 // app.use(express.static())
 app.use(express.json());
-app.use(express.urlencoded());
+app.use(express.urlencoded({ extended: true }));
 
 app.get('/api/menu/:id', (req, res) => {
   const { id } = req.params;
-  res.send(id);
+
+  db.find({ restaurant_id: id }, (err, data) => {
```

Comment:
I noticed you did not state what collection you were querying in. For example i used db.collection('images').find( { id: reqId }). Will yours do the same as my line of code?

Reply:
It does. The 'db' here is a reference to my Menu collection which uses my 'Menu' model and my menuSchema. When I exported it to my server, I just assigned that collection to the 'db' variable.